haystack/preview/components/generators/chat/hugging_face_tgi.py

```diff
@@ -3,14 +3,16 @@
 from typing import Any, Dict, List, Optional, Iterable, Callable
 from urllib.parse import urlparse
 
-from huggingface_hub import InferenceClient
-from huggingface_hub.inference._text_generation import TextGenerationStreamResponse, TextGenerationResponse, Token
-from transformers import AutoTokenizer
-
 from haystack.preview import component, default_to_dict, default_from_dict
-from haystack.preview.components.generators.hf_utils import check_valid_model, check_generation_params
 from haystack.preview.components.generators.utils import serialize_callback_handler, deserialize_callback_handler
 from haystack.preview.dataclasses import ChatMessage, StreamingChunk
+from haystack.lazy_imports import LazyImport
+
+with LazyImport(message="Run 'pip install transformers'") as transformers_import:
+    from huggingface_hub import InferenceClient
+    from huggingface_hub.inference._text_generation import TextGenerationStreamResponse, TextGenerationResponse, Token
+    from transformers import AutoTokenizer
+    from haystack.preview.components.generators.hf_utils import check_valid_model, check_generation_params
```

Comment:
This line is inside this block because `haystack.preview.components.generators.hf_utils` contains Hugging Face utils.
Is it better to do the lazy import in that file?